tests/ops_test.py

```diff
@@ -0,0 +1,57 @@
+# pylint: disable=missing-docstring, line-too-long, protected-access
+import unittest
+from runner import Runner
+
+
+class TestE2E(unittest.TestCase):
+    @classmethod
+    def setUpClass(self):
+        self.snippet = """
+
+            provider "aws" {
+              region = "eu-west-2"
+              skip_credentials_validation = true
+              skip_get_ec2_platforms = true
+            }
+
+            module "ops" {
+              source = "./mymodule"
+
+              providers = {
+                aws = "aws"
+              }
+
+              cidr_block            = "10.2.0.0/16"
+              vpc_subnet_cidr_block = "10.2.1.0/24"
+              az                    = "eu-west-2a"
```

Comment:
need to specify in the environment zone etc so that these can be used in the tags